cmd/hope/utils/nodes_test.go

```diff
@@ -205,7 +213,7 @@ func (s *NodesTestSuite) TestGetNode() {
 	t := s.T()
 	resetViper(t)
 
-	expected := testNodes[5]
+	expected := testNodes[6]
```

Comment:
I don't like how this was needed. Should probably define these nodes by name, and grab them in tests by name, rather than by index.